tests/graph/ahead_behind.c

```diff
@@ -0,0 +1,51 @@
+#include "clar_libgit2.h"
+
+static git_repository *_repo;
+static git_commit *commit;
+static size_t ahead;
+static size_t behind;
+
+void test_graph_ahead_behind__initialize(void)
+{
+    git_oid oid;
+    cl_git_pass(git_repository_open(&_repo, cl_fixture("testrepo.git")));
+
+    git_oid_fromstr(&oid, "be3563ae3f795b2b4353bcce3a527ad0a4f7f644");
+    cl_git_pass(git_commit_lookup(&commit, _repo, &oid));
+}
+
+void test_graph_ahead_behind__cleanup(void)
+{
+	git_commit_free(commit);
+	commit = NULL;
+
+	git_repository_free(_repo);
+	_repo = NULL;
+}
+
+void test_graph_ahead_behind__returns_correct_result(void)
+{
+    
+    git_oid oid;
+    git_oid oid2;
+    git_commit *other;
+
+    git_oid_fromstr(&oid, "e90810b8df3e80c413d903f631643c716887138d");
+    git_oid_fromstr(&oid2, "be3563ae3f795b2b4353bcce3a527ad0a4f7f644");
+    
+    cl_assert_equal_i(0, git_graph_ahead_behind(&ahead, &behind, _repo, &oid, &oid2));
```

Comment:
These should be `cl_git_pass(...)` instead of `cl_assert_equal_i(0, ...)`, they'll emit helpful error messages if anything failed.